jobs/build/gen-assembly/Jenkinsfile

```diff
@@ -57,10 +57,11 @@ node {
                             description: 'Allow matching nightlies built from matching commits but with inconsistent RPMs',
                             defaultValue: false,
                         ),
-                        booleanParam(
-                            name: 'PRERELEASE',
-                            description: 'Prepare the assembly for a prerelease operator advisory. Only valid for preview and candidate assemblies',
-                            defaultValue: false,
+                        string(
```

Comment:
instead of string we can make this 
```
choice(
    name: 'PRE_GA_MODE',
    description: '...',
    choices: [
        ...
    ].join("\n"),
),
```

Reply:
added in https://github.com/openshift-eng/aos-cd-jobs/pull/4075/commits/7383e07cc482c52b9b4751840e0f1476a4ee8c24